otrv4.md

```diff
@@ -1227,10 +1227,12 @@ Bob will be initiating the DAKE with Alice.
 **Alice:**
 
 1. Receives an Identity message from Bob:
-    * Verifies the Identity message as defined in the
-      [Identity message](#identity-message) section.
-    * Picks a compatible version of OTR listed in Bob's profile. If the versions
-      are incompatible, Alice does not send any further messages.
+    * Verifies the Identity message as defined on the
+      [Identity message](#identity-message) section. If the Bob's publickey cannot be
+      verified Alice does not send any further messages.
```

Comment:
Maybe 'if Bob's public keys (`Y` or `B`) are not valid, Alice rejects the message and does not send anything further.

Reply:
Doesn't we have to mention that's Alice who that validate the bob's keys?